saleor/payment/migrations/0016_auto_20200423_0314.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.0.5 on 2020-04-23 08:14
+
+from django.db import migrations
+
+from saleor.plugins.manager import get_plugins_manager
+
+
+def get_plugins():
+    plugins = get_plugins_manager().plugins
+    return {plugin.PLUGIN_NAME: plugin.PLUGIN_ID for plugin in plugins}
+
+
+def change_plugin_name_to_plugin_identifier(apps, schema_editor):
+    plugins = get_plugins()
+    payment = apps.get_model("payment", "Payment")
+
+    for payment in payment.objects.all():
```

Comment:
We should use `iterator` instead of  `all`.

Reply:
@fowczarek is right. This will explode in case of many payment objects in DB.
cc @maarcingebala